### PR TITLE
Fixed all broken links on the platform

### PR DESCRIPTION
**Pull request by BBualdo**
Fixed Links:

FCC Foundation Section:
- Discord Server
- Math Game

Coding Tracker:
- Both Habit Logger links fixed and changed from Habit Tracker to Habit Logger to remain consistent
- Changed Parsing DateTime in C# link to working one from Medium

ShiftsLogger:
- Drinks Info

UnitTesting:
- Replaced 'Importance of Unit Testing' broken link with similar one

Portfolio:
- Replaced 2 example portfolios links with ones I used when looking for inspiration (first one didn't work, second one non subscribing Medium can't read)
- Discord 

Angular Tour of Heroes:
- Fixed broken link (angular.io) is actually deprecated so it's a buggy one

Angular CoffeeTracker:
- Same

React Tic-Tac-Toe:
- Replaced 'Why learn React in 2022' with more fresh link :grin:
- Discord

React Friend Manager:
- Discord

Authentication and Authorization:
- Promised link for Web API Identity Config

Azure: Deploying a full-stack .NET app:
- 'deploy a static website' link
- Movies App link

AZURE FUNCTIONS NEEDS IMPLEMENTATION

Brewery API:
- Register link

### CODE REVIEW

File: TCSA.V2/Helpers/ProjectsSubHelpers/AngularProjectsHelper.cs [hunk 1]
```diff
@@ -55,7 +55,7 @@ internal static List<Project> GetProjects()
 
                 Resources = new List<string>
                 {
-                    "<a target='_blank' href='https://angular.io/tutorial'>Angular Docs: Tour of Heroes Beginner’s Tutorial</a>"
+                    "<a target='_blank' href='https://v17.angular.io/tutorial/tour-of-heroes'>Angular Docs: Tour of Heroes Beginner’s Tutorial</a>"
```
Comment (stevietv) [inline TCSA.V2/Helpers/ProjectsSubHelpers/AngularProjectsHelper.cs:58]: why link to the specific v17 version?

File: TCSA.V2/Helpers/ProjectsSubHelpers/AngularProjectsHelper.cs [hunk 2]
```diff
@@ -100,7 +100,7 @@ internal static List<Project> GetProjects()
                 Resources = new List<string>
                 {
                     "<a target='_blank' href='https://docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-6.0&tabs=visual-studio'>Microsoft Docs: Web Api</a>",
-                    "<a target='_blank' href='https://angular.io/tutorial'>Angular Docs: Tour of Heroes Beginner’s Tutorial</a>",
+                    "<a target='_blank' href='https://v17.angular.io/tutorial/tour-of-heroes'>Angular Docs: Tour of Heroes Beginner’s Tutorial</a>",
```
Comment (stevietv) [inline TCSA.V2/Helpers/ProjectsSubHelpers/AngularProjectsHelper.cs:103]: idem?

File: TCSA.V2/Helpers/ProjectsSubHelpers/AuthProjectsHelper.cs [hunk 3]
```diff
@@ -55,7 +55,8 @@ internal static List<Project> GetProjects()
                 {
                     "<a target='blank' href='https://docs.microsoft.com/en-us/aspnet/core/security/authentication/identity?view=aspnetcore-6.0&tabs=visual-studio'>Introduction to Identity</a>",
                     "<a target='blank' href='https://docs.microsoft.com/en-us/aspnet/core/security/authentication/scaffold-identity?view=aspnetcore-6.0&tabs=visual-studio'>Scaffolding Identity into Existing App</a>",
-                    "<a target='blank' href='https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line'>Logging in C# and .NET</a>"
+                    "<a target='blank' href='https://learn.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line'>Logging in C# and .NET</a>",
+                    "<a target='blank' href='https://learn.microsoft.com/en-us/aspnet/core/security/authentication/identity-api-authorization?view=aspnetcore-8.0'>.NET 8 Web API Identity (for React, Angular, etc.)</a>"
```
Comment (stevietv) [inline TCSA.V2/Helpers/ProjectsSubHelpers/AuthProjectsHelper.cs:59]:
```suggestion
                    "<a target='blank' href='https://learn.microsoft.com/en-us/aspnet/core/security/authentication/identity-api-authorization'>.NET 8 Web API Identity (for React, Angular, etc.)</a>"
```

if we skip the view type then it alwasy defaults to latest .net version

**Review by stevietv — CHANGES_REQUESTED** [review]
nice work! I've made some comments/suggestions

File: TCSA.V2/Helpers/ProjectsSubHelpers/AngularProjectsHelper.cs (hunk 1 above)
Comment (BBualdo) [reply TCSA.V2/Helpers/ProjectsSubHelpers/AngularProjectsHelper.cs:58]: Looked like without it, it doesn't work, but actually it works now 😆 Idk, angular.io after migrating to angular.dev behaves weird

**Pull request closed without merging**